daemon/daemon_unix.go

```diff
@@ -207,6 +207,9 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostC
 		logrus.Warnf("Your kernel does not support kernel memory limit capabilities. Limitation discarded.")
 		hostConfig.KernelMemory = 0
 	}
+	if hostConfig.KernelMemory > 0 && hostConfig.KernelMemory < 4194304 {
```

Comment:
This should be a const and also used here: https://github.com/hqhq/docker/blob/hq_limit_kernel_memory/daemon/daemon_unix.go#L166